ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -23,10 +23,15 @@
     },
     {
       id: 'E6S Ferostorm',
-      regex: Regexes.startsUsing({ source: 'Garuda', id: '4BF[EF]', capture: false }),
+      regex: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Garuda', 'Raktapaksa'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['ガルーダ', 'ラクタパクシャ'], id: ['4BF[EF]', '4C0[45]'], capture: false }),
+      regexCn: Regexes.startsUsing({ source: '迦楼罗', id: '4BF[EF]', capture: false }),
```

Comment:
Can you update the Cn and Ko versions as well?

Reply:
I wish I can but I don't know what is the translation for Raktapaksa in these 2 languages...

Reply:
Could you remove these lines then? I think it's much more obvious that this needs a translation when there's no regex at all.  Having a regex there that's half done seems like something that could easily be missed in the future when the name is known.